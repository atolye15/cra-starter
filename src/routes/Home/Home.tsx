import React, { FunctionComponent } from 'react';
import { Link } from 'react-router-dom';

const Home: FunctionComponent = () => (
  <div>
    <h1>Home page</h1>

    <Link to="/feed">Go to feed</Link>
  </div>
);

export default Home;
