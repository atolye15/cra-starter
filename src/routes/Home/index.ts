import Loadable from 'react-loadable';

import Loading from '../../components/Loading';

const LoadableHome = Loadable({
  loader: () => import('./Home'),
  loading: Loading,
});

export default LoadableHome;
