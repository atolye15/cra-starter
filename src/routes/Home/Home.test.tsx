import React from 'react';
import { shallow } from 'enzyme';

import Home from './Home';

it('renders correctly', () => {
  const wrapper = shallow(<Home />);

  expect(wrapper).toMatchSnapshot();
});
