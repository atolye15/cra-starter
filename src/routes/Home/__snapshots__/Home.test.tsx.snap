// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Home page
  </h1>
  <Link
    to="/feed"
  >
    Go to feed
  </Link>
</div>
`;
